packages/cli/src/__tests__/incomplete-schemas.test.ts

```diff
@@ -92,16 +90,19 @@ describe('[wasm] incomplete-schemas', () => {
         await DbExecute.new().parse(['--file=./script.sql'])
       } catch (e) {
         expect(stripAnsi(e.message)).toMatchInlineSnapshot(`
-          "P1012
-
+          "Prisma schema validation - (get-config wasm)
```

Comment:
Output here changed a bit - we now fail earlier. But the information we output here is the same.